src/main/java/com/dev/cinema/dao/MovieSessionDao.java

```diff
@@ -12,4 +13,6 @@
     MovieSession update(MovieSession session);
 
     void deleteById(Long id);
+
+    Optional<MovieSession> get(Long id);
```

Comment:
```suggestion
    Optional<MovieSession> getById(Long id);
```

Reply:
fixed

Reply:
@Baldinini It's ok to have a `get(Long id)` method name in DAO. In this case, you understand what ID is (the unique identifier)